pkg/policy/cookie/bakery.go

```diff
@@ -31,6 +31,11 @@ type Bakery[C constraints.Unsigned, V comparable] interface {
 	Count() int
 }
 
+// PolicyBakery is a Bakery allocating uint32 cookies for policy baked cookies.
```

Comment:
suggestion: `PolicyBakery is an allocator for uint32 policy cookies`?

saying `PolicyBakery` is a Bakery doesn't help me to understand its purpose :)